marge.nix

```diff
@@ -8,22 +7,12 @@ python.mkDerivation {
   version = "${version}";
   name = "marge-${version}";
   src = ./.;
-  # The dependencies, referring to variables in <nixpkgs>.
-  buildInputs = [py.pylint py.pytest py.pytest-cov];
+  buildInputs = [py.pytest py.pytest-cov py.pytest-pylint py.pytest-runner];
   propagatedBuildInputs = [py.maya py.requests pkgs.openssh pkgs.git];
-  checkPhase = ''
```

Comment:
Hmm, what happened to tests?

Reply:
Never mind, py.pytest-runner. Wonder if we need the PYTHONDONTWRITEBYTECODE.

Reply:
I think is fine